eng/versioning/version_client.txt

```diff
@@ -177,6 +177,7 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-email;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-developer-loadtesting;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-identity-extensions;1.0.0;1.1.0-beta.1
+com.azure:azure-analytics-purview-workflow;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
Let's keep this list alphabetically sorted.

Reply:
Fixed, thanks